docs/fx-core/aad-help.md

```diff
@@ -150,4 +150,55 @@ Root cause of this error is that toolkit can not find Azure AD app with the same
 
 If you can find your Azure AD app, please check your network status and try again.
 
-If you can not find your Azure AD app, please check whether you logged in with the correct account. You can also remove objectId from env.default.json file and then try again.
\ No newline at end of file
+If you can not find your Azure AD app, please check whether you logged in with the correct account. You can also remove objectId from env.default.json file and then try again.
```

Comment:
Seems env.default.json file doesn't exist any more, currently we're using state.xxx.json file